### ISSUE TITLE
Remove gap between "Brave is up to date" & "Version" on brave://settings/help

### ISSUE THREAD

**Issue by Cleveho** [brave/brave-browser#21435]
## Steps to Reproduce

   1. Open Brave
   2. Visit - brave://settings/help

## Actual result:
There is a Gap between "Brave is up to date" & Version.

![Screenshot](https://user-images.githubusercontent.com/48880976/156524024-89c30294-9119-46a8-aab9-e87ca5584282.png)

## Expected result:
Gap between "Brave is up to date" & Version should be removed.

## Reproduces how often: 

Easily reproduced/Always

## Brave version (brave://version info)

Brave	1.36.109 Chromium: 99.0.4844.51 (Official Build) (64-bit) 
Revision	d537ec02474b5afe23684e7963d538896c63ac77-refs/branch-heads/4844@{#875}
OS	Windows 11 Version 21H2 (Build 22000.493)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by GeetaSarvadnya** [comment]
@Cleveho This is as per the design from the beginning, there is no change in it.
cc: @rebron

**Comment by petergeorgas** [comment]
I'd love to take this issue. Currently initializing the repo. Will take a look once it's done.

**Comment by heysujal** [comment]
@petergeorgas are you still working on this issue?
I would like to work on this issue, if you are not working on it

**Comment by heysujal** [comment]
Can anybody guide me how to go about fixing this issue?

**Comment by ancientstraits** [comment]
I would just look for "Brave is up to date" or a variation of it in the soure code @heysujal

**Comment by ancientstraits** [comment]
However, GitHub couldn't find it, and it's a very large codebase, so it will take a lot of time to find it. I will report when done.

**Comment by ancientstraits** [comment]
@heysujal Actually, I found the text in https://github.com/brave/brave-core/blob/5b576ef6125c629d3c876f335a9927f7ca5c208d/app/settings_brave_strings.grdp at line 29:
```xml
<message name="IDS_SETTINGS_UPGRADE_UP_TO_DATE" desc="Status label: Already up to date (Brave)">
    Brave is up to date
</message>
```
We should look for `IDS_SETTINGS_UPGRADE_UP_TO_DATE` in the source code of `brave-core`.

**Comment by ancientstraits** [comment]
Another update: I'm 50% sure the gap is here: https://github.com/brave/brave-core/blob/32061377a37550f542463902e0b396652b86eec8/browser/resources/settings/brave_overrides/settings_menu.js

**Comment by Rixant** [comment]
Can I work on this issue?

**Comment by ancientstraits** [comment]
@bsclifton Good job working on the issue! :+1:

**Comment by bsclifton** [comment]
Whoops- accidentally closed when I did a rebase there. Apologies

@akshay24495 has a fix in https://github.com/brave/brave-core/pull/12849

**Comment by ancientstraits** [comment]
I'm happy it's making it in. :tada:

**Comment by bsclifton** [comment]
Re-opening; seems the changes made with https://github.com/brave/brave-core/pull/12849 didn't work as expected

@akshay24495 would you be up for looking at this again? I suspect one of the following:
1. the solution you had broke due to `unset` (versus `auto`) 
2. The official build code branch is a bit different because local builds don't have the ability to check for updates. You might need to hack that code-path to get a proper test (so that you can see `Brave is up to date` at the same time you see the version)

**Comment by akshay24495** [comment]
Can you say about the hack that code-path to get a proper test ? @bsclifton

**Comment by bsclifton** [comment]
@akshay24495 yeah- I think in the code, you can try and fake the value, so it thinks it's not disabled (or local build)

**Comment by ArshErgon** [comment]
@bsclifton is it still up? can I also work on it?

**Comment by bsclifton** [comment]
@ArshErgon you sure can 😄 If it's open and unassigned, it's up for grabs!

**Comment by ArshErgon** [comment]
@bsclifton do you guys have any channels like discord or telegram?

**Comment by ArshErgon** [comment]
Hey @bsclifton I'm following the steps which are required to install brave locally, I think I am doing something wrong, can you guide me?
my original path: ` /new/brave-browser/`
after that I have to go `/new/brave-browser/src` in `src` I have only one file name .gitkeep, after that I hit `npm install` it work fine, then I do `npm run init` and create a folder name `brave` which only have `.git` and its stuck, its not going on anywhere else showing this `> git clone https://github.com/brave/brave-core.git .`, I have followed this also https://github.com/brave/brave-browser/wiki/Linux-Development-Environment, should I clone `brave-core` directly into the `src` folder?

**Comment by AAYUSH-droid** [comment]
Hey can i work on this issue?!

**Comment by vrindavan** [comment]
Hey, I can also take this up!

**Comment by yashdodani** [comment]
I am looking forward to solve this issue. Thanks for raising this.

**Comment by ancientstraits** [comment]
Wow how did this issue get so much attention

**Comment by JDixon1995** [comment]
Is this issue taken? I'm a beginner, and I'm looking to contribute to open source.

**Comment by SamudraMitra** [comment]
I am new to open source and really want to learn. Please assign me this issue.

**Comment by alpha2207** [comment]
![Resolved on local](https://user-images.githubusercontent.com/73979701/209456115-3335922b-1abf-4a53-8511-6aa1058727d8.png)
I solved this issue on my local. Please assign me this, So that I can contribute.

**Comment by bsclifton** [comment]
@alpha2207 I'll assign - but no need for assignment, just submit the PR and share here 😄

**Comment by spiritanand** [comment]
Can I just make a PR and share here?

**Comment by MohitBansal321** [comment]
This issue still not resolve?

**Comment by alpha2207** [comment]
Nope.

**Comment by MohitBansal321** [comment]
> ![Resolved on local](https://user-images.githubusercontent.com/73979701/209456115-3335922b-1abf-4a53-8511-6aa1058727d8.png) I solved this issue on my local. Please assign me this, So that I can contribute.

but you resolved it?

**Comment by alpha2207** [comment]
I'm having issue setting up the repo on my local.

**Comment by bsclifton** [comment]
I cleared the assignee as it sounds like you ran into an issue @alpha2207 - if someone would like to grab this please do! 
**No assignment required** 😄 

If there is something specific you are running into (not able to submit pull request, problem setting up locally, etc) please share details here and a Brave team member can help you out!

**Comment by Cleveho** [comment]
The issue seems to be resolved on version [Version 1.49.120 Chromium: 111.0.5563.64 (Official Build) (64-bit)](https://brave.com/latest/)

Can anybody confirm please.

**Comment by vassyz** [comment]
Can confirm, it has been fixed 👍
